src/general/OptionsMenu.tscn

```diff
@@ -21,10 +21,16 @@ font_data = ExtResource( 3 )
 
 [sub_resource type="StyleBoxEmpty" id=4]
 
+[sub_resource type="StyleBoxEmpty" id=9]
```

Comment:
It seems that whenever I reopen this in godot editor (after resetting changes) godot absolutely wants to reorder things.
So could you close your godot editor, and reopen it, it should do some reordering on this file.